package.json

```diff
@@ -20,7 +20,8 @@
     "serve-docs": "./node_modules/.bin/gitbook serve",
     "update-translations": "i18next 'app/**/*.js' 'node_modules/tideline/{plugins,js}/**/*.js' 'node_modules/@tidepool/viz/{storiesDatatypes,storybookDatatypes,stories,src}/**/*.js' -c i18next-parser.config.json -o .",
     "storybook": "./node_modules/.bin/start-storybook -p 6006 --ci",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "lint:css":"stylelint './app/components/**/*.js'"
```

Comment:
I'd recommend that we also add the `lint:css` script to the `pretest` script so that a failed css linting will automatically fail the `test` script.